templates/how-to.md

```diff
@@ -0,0 +1,43 @@
+# Title Here
+
+#### Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Step 1](#step-1)
+- [Step 2](#step-2)
+- [Step 3](#step-3)
+- [Step 4](#step-4)
+- [Step 5](#step-5)
+- [Next Steps](#next-steps)
+
+
+### Introduction
+
+This is an introduction to this how to guide. It should be limited to a few
+succinct sentences that introduces what the user will be learning to do. A
+“how to” guide should provide an end-goal, or something that the user will 
+accomplish by following the steps. 
+
+### Prerequisites
```

Comment:
As I recently found with the LDAP solutions guide, it sometimes makes sense to create sub-sections for prerequisites, e.g. 

### Prerequisites

#### Configuration
`foo = bar`

#### Migrations
This guide assumes `KONG_PASSWORD` was set before migrations and that a *super admin* already exists.

#### State
This guide assumes there are no custom *Roles* in the *default Workspace*.

Reply:
so we should offer guidelines on what types of pre-req sections might exist, what they're called, and how they're formatted.

Reply:
good idea

Reply:
Since these are examples, they might belong in the Writing Guide instead of the template. In the template, we could just have 

**Prereq 1**

**Prereq 2**

Reply:
Yeah I am on the fence about how we should do this - I am worried "Prereq 1" and "Prereq2" is too vague and people won't be sure what to put there and we will end up with bolded lists of Prereqs vs bulleted lists

Reply:
It's that vs. them needing to delete all the examples every time.

We could just say, "If needed, divide the Prerequisites into sections with bold titles, e.g.,

**Subsection**"

Reply:
Also, are we going to mention steps within steps, e.g.

## Step 1 - Title for Step 1

1. mini-step
2. mini-step
3. mini-step

## Step 2 - Title for Step 2